### DISCUSSION TITLE
New release(0.12.0)?

### DISCUSSION THREAD

**Discussion by YasirAsar** [elixir-protobuf/protobuf#342] [Q&A] [answered]
Hey, I see you've added support for Any type. I like to use it. Do you have any idea like when the next release is?

**Comment by whatyouhide** [comment, marked as answer]
I don't have time right now to make a release of Protobuf, unless sponsored by someone to dedicate some time to it. I think this is a few months away at this moment.

**Reply by YasirAsar** [reply]
Thanks for the quick reply
